pkg/controller/ads/ads_processor.go

```diff
@@ -326,6 +320,7 @@ func (p *processor) Reset() {
 		return
 	}
 	p.lastNonce = &lastNonce{}
+	p.Cache = NewAdsCache()
```

Comment:
I donot think we should reinit a cache, because in ads handler, we depend on the cache to do optimization,

Ycan reset p.Cache.edsClusterNames and routeNames